testsuite/dynaconf_loader.py

```diff
@@ -109,6 +123,17 @@ def load(obj, env=None, silent=None, key=None):
             server_url=ocp_setup.get("server_url"),
             token=ocp_setup.get("token"))
 
+        apicast_ocp_setup = \
+            weakget(obj)["threescale"]["gateway"]["OperatorApicast"]["openshift"]["servers"]["default"] % ocp_setup
+
+        apicast_project = \
+            weakget(obj)["threescale"]["gateway"]["OperatorApicast"]["openshift"]["project_name"] % "apicast-ga"
```

Comment:
@pehala 
Is this the right place to specify openshift server for apicast operator?
This should be in default configs set up to `apicast-ga` or `apicast-operator` so I can change this to normal lookup?

Reply:
The best way would be probably something like:
```
client = ocp
apicast_section = weakget(obj)["threescale"]["gateway"]["OperatorApicast"]["openshift"] % None
if apicast_section:
  client = OpenshiftClient(**apicast_section)
```